pom.xml

```diff
@@ -1056,6 +1094,81 @@
     </profile>
 
     <profile>
+      <id>jacoco</id>
```

Comment:
We are going to use ```jacoco``` profile in Travis CI config as an addition check, and I think we will migrate to JaCoCo completely.